ghost/core/core/frontend/src/ghost-stats/ghost-stats.bundled.js

```
@@ -0,0 +1,545 @@
+          payload: processedPayload
+        })
+      );
+    }
+    function _trackPageHit() {
+      if (window.__nightmare || window.navigator.webdriver || window.Cypress)
+        return;
+      let country, locale;
+      try {
+        const timezone = Intl.DateTimeFormat().resolvedOptions().timeZone;
+        country = timezoneData[timezone];
+        locale = navigator.languages && navigator.languages.length ? navigator.languages[0] : navigator.userLanguage || navigator.language || navigator.browserLanguage || "en";
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)